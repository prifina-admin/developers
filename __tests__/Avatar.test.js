import React from "react";
import renderer from "react-test-renderer";
import Avatar from "../src/Avatar";
import ThemeProvider from "../src/theme/ThemeProvider";

test("Avater renders initials correctly", () => {
    const tree = renderer.create(<ThemeProvider>
        <Avatar initials={"TR"} width={100} />
    </ThemeProvider>).toJSON();
    expect(tree).toMatchSnapshot();
});

test("Avater renders image correctly", () => {
    const tree = renderer.create(<ThemeProvider>
        <Avatar src={"https://i.pravatar.cc/100"} alt={"photo"} width={100} />
        </ThemeProvider>).toJSON();
    expect(tree).toMatchSnapshot();
});
