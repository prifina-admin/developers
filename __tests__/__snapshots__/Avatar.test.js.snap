// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avater renders image correctly 1`] = `
<div
  className="ThemeProvider__Base-sc-1rk52n4-0 emGfPA"
>
  <img
    alt="photo"
    className="Image__ImageElement-sc-1qpr0nl-0 eYGcjW"
    height="auto"
    src="https://i.pravatar.cc/100"
    width={100}
  />
</div>
`;

exports[`Avater renders initials correctly 1`] = `
<div
  className="ThemeProvider__Base-sc-1rk52n4-0 emGfPA"
>
  <div
    className="Box__BoxElement-sc-1iw4lg9-0 dRQcpO Avatar__InitialsElement-unjslo-0 cLdDIf"
  >
    <span
      className="Text__TextElement-sc-102lho-0 cOhPIL"
    >
      TR
    </span>
  </div>
</div>
`;
